packages/notifi-core/lib/models/WalletBlockchain.ts

```diff
@@ -10,4 +11,6 @@ export type WalletBlockchain =
   | 'NEAR'
   | 'OPTIMISM'
   | 'INJECTIVE'
-  | 'SUI';
+  | 'SUI'
+  | 'OSMOSIS'
+  | 'EVMOS';
```

Comment:
It might be better not to include the unsupported chains:
- OFF_CHAIN
- EVMOS
- OSMOSIS